### PR TITLE
CB-4 Team Component

### PR DESCRIPTION
**Pull request by katy-rose**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
This PR adds in the feature of the team component

* **What is the current behavior?What does the code do before you made the new changes?** 
Currently the main branch does not contain this component and the landing page is empty

* **What is the new behavior?**
This adds in 2 components to manage the section that describes individual team members. The component shows each team member in a separate reactstrap card.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
No

* **Other information**:

### CODE REVIEW

**Review by Codubee — CHANGES_REQUESTED** [review]
W O W Katy this is fantastic! Great work on this really this is good. Very organized code! Only one change to request and that would be to rename the css file to just TeamComponent.css. The .css ending gives it away to the reading that its a stylesheet. Other than that though this is really good. Love the colored background on these as well!

File: frontend/src/components/LandingPage/TeamComponent/TeamCard.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Card, CardBody, CardTitle, CardText, CardLink, 
+        CardImg, CardImgOverlay } from 'reactstrap';
+import spaceBG from './space.jpg';
+import GitHub from './whiteGithub.png'
+import './TeamComponentCSS.css'
```
Comment (Codubee) [inline frontend/src/components/LandingPage/TeamComponent/TeamCard.js]: With the new filename change, I had to change the file name here.

File: frontend/src/components/LandingPage/TeamComponent/TeamComponent.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { Row, Col } from 'reactstrap';
+import './TeamComponentCSS.css'
```
Comment (Codubee) [inline frontend/src/components/LandingPage/TeamComponent/TeamComponent.js]: With the new filename change, I had to change the file name here as well.

**Review by Codubee — CHANGES_REQUESTED** [review]
Hi Katy I eft some comments about the css file name change. One thing I didnt see before either is the location of the images. The public folder is usually where the images go and importing them would work like [this](https://github.com/Codubee/SpaceJunkTTH2-4/blob/ad2164e09734a905401f879e312de3ecca85152d/frontend/src/App.js#L1). What do you think about moving the images to the public folder?

**Review by Codubee — CHANGES_REQUESTED** [review]
Great work Katy. Just one comment can we move the images to the public folder. Its located in the frontend folder just outside of the src folder.

**Comment by katy-rose** [comment]
> Great work Katy. Just one comment can we move the images to the public folder. Its located in the frontend folder just outside of the src folder.

I moved images to the public folder, but received an error stating the files could not be outside the src folder, so I just moved them to src. I'll try again and screenshot what exact error I am receiving.

**Comment by katy-rose** [comment]
This is the error I receive when running npm start when the pictures are in the public folder:
"Module not found: You attempted to import ../../../../public/space.jpg which falls outside of the project src/ directory. Relative imports outside of src/ are not supported."

Maybe I'm importing incorrectly?
image

**Comment by Codubee** [comment]
Ah I pointed you in the wrong direction. Sorry about that Katy. When importing from the public folder we can set a const equal to the path like [this](https://github.com/Codubee/ExploreCitiesTTH4-6/blob/ca8f6502b4be75500f177f490c4027c24663b3cf/frontend/src/components/LandingPage/Tools/Tools.js#L8). Try that out and see if it works. If it doesnt lets discuss it at our next meeting.

**Review by Codubee — APPROVED** [review]
LGTM!

**Pull request merged**